dockerfiles/centos-7-pg10/Dockerfile

```diff
@@ -19,6 +19,18 @@ RUN [[ centos != oraclelinux ]] || [[ 7 != 7 ]] || ( \
             ol7_optional_archive \
         && yum clean all )
 
+
+RUN if [[ centos   == oraclelinux ]] && [[ 7 == 7 ]]; then yum install -y wget \
+        && wget http://mirror.centos.org/centos/7/os/x86_64/Packages/lz4-devel-1.8.3-1.el7.x86_64.rpm \
+        && rpm -ivh lz4-devel-1.8.3-1.el7.x86_64.rpm ; \
+        elif [[ centos   == oraclelinux ]] && [[ 7 == 6 ]]; then yum install -y wget \
+        && wget https://cbs.centos.org/kojifiles/packages/lz4/r131/1.el6/x86_64/lz4-r131-1.el6.x86_64.rpm \
+        && rpm -ivh lz4-r131-1.el6.x86_64.rpm \
+        && wget https://cbs.centos.org/kojifiles/packages/lz4/r131/1.el6/x86_64/lz4-devel-r131-1.el6.x86_64.rpm \
+        && rpm -ivh lz4-devel-r131-1.el6.x86_64.rpm;  \
+        else yum install -y lz4 lz4-devel; fi
```

Comment:
could use a comment why we are installing it like this instead of adding the packages to the statement below. I guess not all centos verions do support the packages? Could this become an issue for our releases?

Reply:
Ok I'm adding comment